contracts/MockAcrossMessageContract.sol

```diff
@@ -0,0 +1,40 @@
+//SPDX-License-Identifier: Unlicense
+pragma solidity ^0.8.0;
+
+// This interface is expected to be implemented by any contract
+// that expects to receive messages from the SpokePool.
+// Retrieved from https://github.com/across-protocol/contracts-v2/blob/master/contracts/SpokePool.sol
+interface AcrossMessageHandler {
+  function handleAcrossMessage(
+    address tokenSent,
+    uint256 amount,
+    bool fillCompleted,
+    address relayer,
+    bytes memory message
+  ) external;
+}
+
+contract MockAcrossMessageContract is AcrossMessageHandler {
+  function handleAcrossMessage(
+    address tokenSent,
+    uint256 amount,
+    bool fillCompleted,
+    address relayer,
+    bytes memory message
+  ) external view override {
+    // "Use" the variables to avoid a compilation warning.
+    tokenSent;
```

Comment:
The more correct fix to this is to remove the variable name in the function declaration. For example instead of declaring:

```sol
function x(uint256 unusedVar, ...)
```

change it to
```sol
function x(uint256, ...
```